fe1-web/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "experimentalDecorators": true,
```

Comment:
Is this needed, and for what ?

Reply:
I don't remember where it came from, but the code works well without it. I will delete it.

Reply:
I think WebStorm was possibly complaining without it 😅 I remember having a warning like that.